src/fsharp/CheckComputationExpressions.fs

```diff
@@ -1283,14 +1283,13 @@ let TcComputationExpression cenv env (overallTy: OverallTy) tpenv (mWhole, inter
                     // Build the 'Bind' call
                     Some (transBind q varSpace bindRange "Bind" [mergedSources] consumePat letSpBind innerComp translatedCtxt)
 
-        | SynExpr.Match (spMatch, expr, clauses, m) ->
-            let mMatch = match spMatch with DebugPointAtBinding.Yes mMatch -> mMatch | _ -> m
+        | SynExpr.Match (matchKeyword=mMatch ;matchDebugPoint=spMatch; withKeyword=mWith; expr=expr; clauses=clauses; range=m) ->
```

Comment:
Oh I see, it's too intrusive I think, I'm not in favour of this.

Please only switch to this style for matches that are majority `_`

Reply:
It only affects a few places in the PR, I'll point them out